### DISCUSSION TITLE
Rename Build

### DISCUSSION THREAD

**Discussion by Malix-Labs** [lisonge/vite-plugin-monkey#52] [Q&A] [answered]
Hello @lisonge!

Thanks for your awesome work on this plugin, I really do appreciate it.

## Question

I noticed the build file is being named after the `package.json` `name` value.
Is there a way to put an option in the `vite.config` file for it to be customized?

Thanks a lot!

**Comment by lisonge** [comment, marked as answer]
MonkeyOption.build.fileName

<https://github.com/lisonge/vite-plugin-monkey/blob/5e66e591f1b0e884568d734237da3894aaf11eb7/packages/vite-plugin-monkey/src/node/types.ts#L174-L180>

**Reply by Malix-Labs** [reply]
Thanks a lot!

Should this file be considered as the official documentation?

**Reply by lisonge** [reply]
> Should this file be considered as the official documentation?

official documentation is [README.md#config](https://github.com/lisonge/vite-plugin-monkey#config)

you can also find the configuration of fileName in [README.md#config](https://github.com/lisonge/vite-plugin-monkey#config)
